### PR TITLE
changes for cob4-13 

### PR DESCRIPTION
**Pull request by HannesBachter**

### CODE REVIEW

File: cob_bringup/robots/cob4-16.xml [hunk 1]
```diff
@@ -134,14 +134,14 @@
 		<include file="$(find cob_bringup)/components/canopen_generic.launch">
 			<arg name="robot" value="$(arg robot)"/>
 			<arg name="component_name" value="arm_left"/>
-			<arg name="can_device" value="can2"/>
+			<arg name="can_device" value="can3"/>
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-16.xml]: `cob4-16` has been shipped
thus, please undo these changes

File: cob_hardware_config/robots/cob4-13/urdf/cob4-13.urdf.xacro [hunk 2]
```diff
@@ -30,6 +30,7 @@
   <!-- sensors -->
   <xacro:include filename="$(find cob_description)/urdf/sensors/realsense.urdf.xacro" />
   <xacro:include filename="$(find cob_description)/urdf/sensors/realsense_zr300.urdf.xacro" />
+  <xacro:include filename="$(find cob_description)/urdf/sensors/kinect.urdf.xacro" />
```
Comment (fmessmer) [inline cob_hardware_config/robots/cob4-13/urdf/cob4-13.urdf.xacro]: remove this line - there is no kinect mounted to the robot

**Review by fmessmer — CHANGES_REQUESTED** [review]
as the head is fixed, plz remove all `head_*.yaml` in `cob_hardware_config/robots/cob4-13/config`

**Review by fmessmer — APPROVED** [review]
@HannesBachter 
I added a few commits to fix my remarks...I also tested the simulation (had to add the gripper part again for cob4-13 and cob4-16)

Please test again on HW after @ipa-tif fixed the HW-Issues

**Comment by HannesBachter** [comment]
tested on HW - works as expected

**Comment by fmessmer** [comment]
I merged in order to start playing back the config improvements from unity_robots

**Pull request merged**